modules/mobile/spec/request/appointments_vaos_v2_list_request_spec.rb

```diff
@@ -154,6 +154,22 @@
         # removed vetextId, typeOfCare, and friendlyLocationName due to change in behavior decided for v2 adaption to v0
         expect(response_v2).to eq(response_v0)
       end
+
+      it 'has access and returned va appointments having partial errors' do
+        expect do
+          VCR.use_cassette('appointments/VAOS_v2/get_appointment_200_partial_error',
+                           match_requests_on: %i[method uri]) do
+            get '/mobile/v0/appointments', headers: iam_headers, params:
+          end
+
+          expect(response).to have_http_status(:ok)
```

Comment:
I would also suggest testing the `appointments.count` to ensure that we're still receiving appointments, just to confirm we didn't somehow screw up that logic.

Reply:
added